backend/data_distro/api_tests/schema/auditor.jsonnet

```diff
@@ -0,0 +1,26 @@
+local Base = import 'APITestBase.libsonnet';
+local Types = Base.Types;
+
+Types.object {
+  properties: {
+    id: Types.integer,
+    cpa_phone: Types.integer, // should this be a string?
```

Comment:
@eazapata23 there are custom types for a lot of these "contact info" fields that will need to be added and verified.